build.gradle

```diff
@@ -1,14 +1,20 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
+    def kotlinVersion = "1.1.51"
+
     repositories {
         jcenter()
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:3.0.0'
         classpath 'com.neenbedankt.gradle.plugins:android-apt:1.8'
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion"
     }
 
+    // Kotlin
+    ext.kotlinLib = "org.jetbrains.kotlin:kotlin-stdlib-jre7:$kotlinVersion"
```

Comment:
Since this is using the 3.0.0 build tools you can use jre8.

reference: https://developer.android.com/studio/write/java8-support.html